### PR TITLE
dnsproxy: print total number of rules if too many

### PR DESCRIPTION
**Pull request by kkourt**
GetRules() will not process more than 1000 rules per port.
Print how many are the total rules in the message.

Signed-off-by: Kornilios Kourtis <kornilios@isovalent.com>

### CODE REVIEW

File: pkg/fqdn/dnsproxy/proxy.go
Comment (christarazi) [inline pkg/fqdn/dnsproxy/proxy.go]: Would it make more sense as a log field, rather than a formatted string?

Nit: we can also inline `total` to avoid an extra var by just using `len(...)`

**Review by christarazi — APPROVED** [review]

File: pkg/fqdn/dnsproxy/proxy.go
Comment (kkourt) [reply pkg/fqdn/dnsproxy/proxy.go]: Thanks, Chris. Pushed a new version with these two changes.

**Comment by kkourt** [comment]
Please do not merge until #13992  to avoid conflicts for the latter. Once #13992 is merged, I'll rebase this one.

**Review by jibi — APPROVED** [review]

**Comment by kkourt** [comment]
test-me-please

**Pull request merged**
